<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_Estoque</class>
 <widget class="QDialog" name="Fm_Estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QTabWidget" name="tw_main">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_addProd">
      <attribute name="title">
       <string>Adicionar Produto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lbl_prodCod">
             <property name="text">
              <string>Cod. Produto</string>
             </property>
             <property name="buddy">
              <cstring>le_prodCod</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_prodCod"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_prodName">
             <property name="text">
              <string>Nome do Produto</string>
             </property>
             <property name="buddy">
              <cstring>le_prodName</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_prodName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lbl_prodDesc">
             <property name="text">
              <string>Descrição do Produto</string>
             </property>
             <property name="buddy">
              <cstring>le_prodDesc</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_prodDesc"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="lbl_prodQtt">
               <property name="text">
                <string>Quantidade em Estoque</string>
               </property>
               <property name="buddy">
                <cstring>le_prodQtt</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_prodQtt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="lbl_bValue">
               <property name="text">
                <string>Valor de Compra</string>
               </property>
               <property name="buddy">
                <cstring>dsb_bValue</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dsb_bValue"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="lbl_sValue">
               <property name="text">
                <string>Valor de Venda</string>
               </property>
               <property name="buddy">
                <cstring>dsb_sValue</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dsb_sValue"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="lbl_supplier">
               <property name="text">
                <string>Fornecedor</string>
               </property>
               <property name="buddy">
                <cstring>cb_supplier</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cb_supplier"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btt_add">
             <property name="text">
              <string>Adicionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btt_clear">
             <property name="text">
              <string>Limpar</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_manegeProd">
      <attribute name="title">
       <string>Gerenciamento de produtos</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGroupBox" name="gb_filtros">
             <property name="title">
              <string>Filtros:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="rb_nome">
                <property name="text">
                 <string>Nome</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_descricao">
                <property name="text">
                 <string>Decrição</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="lbl_pesquisa">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Pesquisa:</string>
               </property>
               <property name="buddy">
                <cstring>le_pesquisa</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_pesquisa"/>
             </item>
             <item>
              <widget class="QPushButton" name="btt_reload">
               <property name="text">
                <string>Recarregar</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tw_produtos">
           <column>
            <property name="text">
             <string>Cod. Produto</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nome</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Valor - Compra</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Valor - Venda</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estoque</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="btt_editar">
             <property name="text">
              <string>Editar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btt_excluir">
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
